workflow/schemas/units.schema.yaml

```diff
@@ -23,6 +23,9 @@ properties:
   adapters:
     type: string
     description: cutadapt adapter trimming settings to use (see https://cutadapt.readthedocs.io)
+  umis:
+    type: string
+    description: path to FASTQ file containing read names and corresponding molecular barcodes (leave emtpy if not exising)
```

Comment:
Shouldn't there also be an update to config.schema.yaml, for the filter changes?